itchef/cookbooks/fb_helpers/libraries/node_methods.rb

```diff
@@ -120,7 +120,7 @@ def fedora_family?
     end
 
     def macos?
-      self['platform'] == 'mac_os_x'
+      self['platform'] == 'mac_os_x' || self['platform'] == 'macos'
```

Comment:
We're still going to present the platform as mac_os_x in Ohai. This isn't the way you want to do this.

Reply:
If the intention is for 'platform' to return `mac_os_x` then you're going to have to mask it, because Ohai is going to get this wrong in 10.16.

```
sw_vers -productName
macOS
```

Reply:
https://github.com/chef/ohai/blob/master/lib/ohai/plugins/darwin/platform.rb#L26-L30 will need to be changed to return 'mac_os_x' instead of just taking what `sw_vers` gives.

Reply:
Fixed here: https://github.com/chef/ohai/pull/1470